docs/AdminGuide.md

```diff
@@ -108,3 +108,63 @@ Example entry:
     "time": "2021-10-05 14:44:59",
 }
 ```
+
+## Update the expiry date of mismatches
+
+1. SSH into toolforge.
+
+    ```bash
+    ssh <your_username>@login.toolforge.org
+    ```
+
+1. The `become` commands allows you to sudo to another user while retaining your personalized environment.
+
+    ```bash
+    become mismatch-finder
+    ```
+
+1. Get into the database using the toolforge alias for accessing Mariadb 
```

Comment:
This step should be removed now (it was replaced by the next two steps).

Reply:
true!